package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-featureflow-client",
-  "version": "1.3.6",
+  "version": "17.0.0",
```

Comment:
is that right? I though you were upgrading to react 17, not upgrading this version of ff to 17?

Reply:
I thought it might be good to indicate the intended react major. Perhaps not, its implied in the deps of course.